timeclock.py

```diff
@@ -158,11 +174,10 @@ def create_dev_users():
 
 
 @app.cli.command("create_health_screen_users")
-def create_dev_users():
+def create_health_screen_users():
```

Comment:
The old name was used twice, once here and once in the fucntion preceding this one